test/behaviour/typeql/BUILD

```diff
@@ -42,7 +42,10 @@ host_compatible_java_library(
     ],
     native_libraries_deps = [
         "//concept:concept",
+        "//common:common",
         "//common/test:util",
+        "//traversal:traversal",
+        "//pattern:pattern",
```

Comment:
to test all permutations of `match` queries in BDD, we need to access both `Traversal` and `Pattern`